addon/components/base/bs-form/element/control.js

```diff
@@ -16,6 +16,13 @@ export default Ember.Component.extend({
   value: null,
 
   /**
+   * @property ariaDescribedby
+   * @type {string}
+   * @public
+   */
+  ariaDescribedby: null,
```

Comment:
But you're overriding the emitted attribute name anyway, so I'd stick with convention in the attribute naming.